src/monthlychallenges/oct-2021/index.njk

```diff
@@ -0,0 +1,71 @@
+---
+title: 'Monthly Theme & Challenge for October, 2021: HacktoberFest!'
+description: 'October challenge -> Participate to Hacktoberfest! This month, our members will participate to the DigitalOcean HacktoberFest by being maintainers, contributors, or mentors.'
+layout: layouts/simple.njk
+date: 2021-10-01
+tags: monthlychallenges
+---
+
+
+<h1>
+  <small>Monthly Challenge for October, 2021:</small>
+It's HacktoberFest: participate to Open Source, learn and have fun!</h1>
+
+<p class="lead">
+ This month, we have three tracks. Maintainers will provide issues labeled for HacktoberFest. Contributors will solve issues, and can get support from Mentors. Mentors will help contributors and why not maintainers depending on their area of expertise.
+</p>
+
+<p><strong>Challenge Team Leads & Facilitators</strong>: Aurelie Verrot & Andrew Bush</p>
+
+<hr/>
+
+<h2>Theme</h2>
+<p>The 2021 HacktoberFest!</p>
+
+<h2>Challenge</h2>
+
+<strong>Maintainers</strong>
+<p>.</p>
+
+<strong>Contributors</strong>
+<p>.</p>
+
+<strong>Mentors</strong>
+<p>.</p>
+
+<h2>How to Participate</h2>
+<h4>Before starting</h4>
+<ol>
+  <li>
+    If you are a contributor or a maintainer, make sure to sign up to the <a href="https://hacktoberfest.digitalocean.com/">HacktoberFest official website</a>.
+  </li>
+  <li>
+    Also if you haven't already, sign up to our dashboard: <a href="https://hacktoberfest.virtualcoffee.io/">HacktoberFest Initiative</a>.
+  </li>
+</ol>
+
+<h4>Weekly check-ins</h4>
+<p> Every Friday of the month, come to our Zoom meeting for a check in and share your progress, ask questions, find help and support. To join, find the link in our #monthly-challenge Slack channel.</p>
+
+<h4>How do I share my progress?</h4>
+<p>Share your progress in the #monthly-challenge channel in Slack. While no other platform is imposed, it can be a good idea to also share on social media for more reach but only if you are comfortable to do so (Twitter using -or not- the hashtag #VCMonthlyChallenge, a personal blog, a post on
+  <a href="https://dev.to/">DEV.to</a>, you get the idea)</p>
```

Comment:
```suggestion
<p>
  Share your progress in the #monthly-challenge channel in Slack. While no other platform is imposed, it can be a good idea to also share on social media for more reach but only if you are comfortable doing so.  You can share on Twitter using -or not- the hashtag #VCMonthlyChallenge, a personal blog, a post on <a href="https://dev.to/">DEV.to</a>, you get the idea!
</p>
```
Some reformatting and sentence structure suggestions.